packages/react-composites/tests/browser/chat/app/index.tsx

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import React, { useState, useEffect } from 'react';
+import ReactDOM from 'react-dom';
+
+import { ChatAdapter, createAzureCommunicationChatAdapter, ChatComposite } from '../../../../src';
+
+import { createUserAndThread } from './identity';
+
+const connectionString = process.env.CONNECTION_STRING;
```

Comment:
Is this running in browser code?
So dotnet-webpack plugin is parsing the whole env object to the browser stack, interesting learning

Reply:
yes. :D